.github/cliff.toml

```
@@ -61,14 +61,14 @@ Full commit diff: [`{% if previous.version -%}
 {%- endif %}...{{ last_commit }}`][{{ this_version }}]
 {% if github.contributors | filter(attribute="is_first_time", value=true) | length != 0 %}
   ## New Contributors
-{%- endif -%}
+    {% for contributor in github.contributors | filter(attribute="is_first_time", value=true) %}
+    - @{{ contributor.username }} made their first contribution
```

restore escapes from tagging user names
```suggestion
    - \\@{{ contributor.username }} made their first contribution
```